pom.xml

```diff
@@ -2117,7 +2119,7 @@
               <include>**/*Suite.java</include>
             </includes>
             <reportsDirectory>${project.build.directory}/surefire-reports</reportsDirectory>
-            <argLine>-ea -Xmx4g -Xss4m -XX:ReservedCodeCacheSize=${CodeCacheSize}</argLine>
+            <argLine>-ea -Xmx3g -Xss4m -XX:ReservedCodeCacheSize=${CodeCacheSize} ${jdk11.module.args}</argLine>
```

Comment:
Hi, @squito . Do we need to reduce this back to `3g` for JDK11?

Reply:
oh no, sorry that was not supposed to be part of this change, sorry.